packages/postgresql-node-connector/test/save-load.spec.ts

```diff
@@ -22,6 +22,12 @@ describe("Postgres store service implementation behavior adheres to interface",
     );
   });
 
+  afterAll(async () => {
+    if (postgresService.reset !== undefined) {
```

Comment:
In the tests this should always be defined, so if it is in fact undefined we should throw an error.